scripts/ci/test_plan.py

```diff
@@ -149,7 +149,7 @@ def get_plan(self, options, integration=False, use_testsuite_root=True):
             os.remove(fname)
 
     def find_modules(self):
-        if 'west.yml' in self.modified_files:
+        if 'west.yml' in self.modified_files and args.commits is not None:
```

Comment:
I think `find_modules` should work regardless if the modified files were given explicitly or the script found them with `git.diff`. Therefore, I would change a logic in L:155

Reply:
When list of files is provided, then there is no commit (base) to compare with current HEAD.
There is no clear rule how to get that base revision.